app/Importers/DataImporter.php

```diff
@@ -48,6 +49,13 @@ public static function getProviderName(): string
         return $classNameParts[0];
     }
 
+    public function translate(string $word, $language): string
+    {
+        $translate = new GoogleTranslate($language);
```

Comment:
Could you inject it to the constructor?

Reply:
sure, done

Reply:
It creates a problem with ImporterTest - GoogleTranslate is a final class, it can't be mocked

Reply:
done